src/app/templates/SeedPhraseInput/SeedLengthSelect/SeedLengthSelect.tsx

```diff
@@ -3,8 +3,10 @@ import React, { FC, useCallback, useEffect, useRef, useState } from 'react';
 import classNames from 'clsx';
 
 import { ReactComponent as SelectArrowDownIcon } from 'app/icons/select-arrow-down.svg';
+import { setTestID } from 'lib/analytics';
 import { t } from 'lib/i18n';
 
+import { ImportAccountSelectors } from '../../../pages/ImportAccount/selectors';
```

Comment:
```suggestion
import { ImportAccountSelectors } from 'app/pages/ImportAccount/selectors';
```
Please don't forget to sort imports